master/usage/assigning-ip-pools-per-node.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Assigning IP pools per-Node
+---
+
+## About IP pool Node selection
+
+Starting from {{site.prodname}} {{site.min-versions.ip-pool-node-select}}, IP pools can be configured 
+to restrict IP address assignment of workloads to specific Nodes. This feature is useful for
+network/system administrators that are interested in assigning IPs to pods based on rack affinity.
+This guide walkthroughs an example of how to setup and manage this feature.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Labeled Nodes**
+
+In order to assign IP pools to specific Nodes, these Nodes must be labeled. See the documentation for [calicoctl label]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands/label) and [kubectl label](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/#add-a-label-to-a-node) for more information on how to do this. We recommend using `kubectl label`.
+
```

Comment:
I wondder if we should have a high level "How this works" type paragraph before diving into the example, where we can talk about the precedence given to IP pools vs annotations, etc.